content/blog/2018/08/2018-08-15-contributor-summit-nice.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Join us at the Jenkins Contributor Summit Nice, Tuesday 23 October 2018"
+tags:
+- community
+- events
+- jenkins-world
+author: tracymiranda
+---
+
+image::/images/conferences/devops-world-cs-2018.jpg[role=right]

```

Comment:
This image is for the contributor summit in SF.

Reply:
Yes. Alyssa is sorting a new one so will get that updated once ready